app/presenters/vendor_api/base.rb

```diff
@@ -53,7 +53,14 @@ def version_available_in_environment?(version)
     end
 
     def cache_key(model, api_version, suffixes = {})
-      CacheKey.generate("#{api_version}_#{model.cache_key_with_version}#{suffixes.hash}")
+      suffixes_name = suffixes.values.compact_blank.join('-')
+
+      [
+        'vendor_api',
+        api_version,
+        model.cache_key_with_version,
+        suffixes_name,
+      ].compact_blank.join('-')
```

Comment:
In the PR description, would you mind adding an example of what this looks like (before and after), and a brief rationale?